src/FirmwarePlugin/PX4/MavCmdInfoFixedWing.json

```diff
@@ -59,9 +59,16 @@
             "param3": {
                 "label": "Radius:",
                 "units": "m",
-                "default": 100,
+                "default": 50,
                 "decimalPlaces": 0
             },
+            "param4": {
+                "label":            "Next waypoint start:",
```

Comment:
A bit concerned about such a long label fitting on all form factors. I'm going to have to merge and try on a phone to see.